resume-builder/README.md

```diff
@@ -0,0 +1,68 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
I like how extensive the README.md is. Really useful when you first stumble upon a repo and this breakdown really makes exploring through easier to understand.